core/src/main/resources/lib/form/password.jelly

```diff
@@ -61,12 +61,60 @@ THE SOFTWARE.
     </st:attribute>
   </st:documentation>
   <f:prepareDatabinding />
-  <m:input xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary"
-         class="setting-input ${attrs.checkUrl!=null?'validated ':''}${attrs.clazz}"
-         name ="${attrs.name ?: '_.'+attrs.field}"
-         value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"
-         type="password"
-         checkMethod="post"
-         ATTRIBUTES="${attrs}" EXCEPT="field clazz value" />
+  <j:set var="value" value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"/>
+  <j:choose>
+    <j:when test="${ value != null }">
+      <st:adjunct includes="lib.form.password.password"/>
+      <div class="hidden-password">
+        <m:input xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary"
+                 class="hidden-password-field setting-input ${attrs.checkUrl!=null?'validated ':''}${attrs.clazz}"
+                 name ="${attrs.name ?: '_.'+attrs.field}"
+                 value="${value}"
+                 type="hidden"
+                 checkMethod="post"
+                 ATTRIBUTES="${attrs}" EXCEPT="field clazz value" />
+        <div class="hidden-password-placeholder">
+          <div class="hidden-password-legend">
+            <svg width="20px" height="25px" viewBox="0 0 25 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Come to think of it, I think we don't need to inline this svg anymore. It was inlined in the standalone library because there was no good way to reference a static asset from a jar. This could be moved to a more appropriate location in jenkins to be a reusable static asset.